contents/docs/deployment/vpc/vpc-aws.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Virtual Private Cloud on AWS
+sidebar: Docs
+showTitle: true
+---
+
+_[Click here for an overview of what a PostHog Virtual Private Cloud is](/docs/deployment/vpc/vpc)_
+
+To get started with a Virtual Private Cloud on AWS, you'll need to give us access keys to an IAM user with the right permissions.
+
+## 1. Go to IAM in your AWS console
+![AWS add user](../../../images/vpc-aws-add-user.png)
+
+## 2. Create a new IAM user
+
+It makes sense to make it `companyname-posthog`.
+
+![AWS create user](../../../images/vpc-aws-create-user.png)
+
+
+## 3. Create a new permissions group
+
+This way you'll be able to scope down what we have access to heavily.
+
+The permissions we need are:
+```json
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Sid": "PosthogDeploy",
+            "Effect": "Allow",
+            "Action": [
+                "rds:*",
+                "s3:*",
+                "ec2:*",
+                "elasticloadbalancing:*",
+                "eks:*",
+                "elasticache:*"
+            ],
+            "Resource": "*"
+        }
+    ]
+}
+```
```

Comment:
We will almost certainly get push back on these permissions. I need to figure out the best way to scope permissions to only resources created by an IAM account and the docs are really thick. Going to take a bit of research.